templates/test/ci/cluster-template-prow-ci-version-dual-stack.yaml

```diff
@@ -124,6 +124,26 @@ spec:
         set -o errexit
         [[ $(id -u) != 0 ]] && SUDO="sudo" || SUDO=""
 
+        echo "Use OOT credential provider"
+        mkdir -p /var/lib/kubelet/credential-provider
+        # Update the link below
```

Comment:
Actually, this is a comment that should be removed but I forget to in our customized template...

Reply:
removed